src/Robo/Plugin/Commands/ContainerCommands.php

```diff
@@ -41,9 +65,25 @@ public function containersExec($container, $execute_command) {
       ->run();
   }
 
+  /**
+   * Open a SSH connection to specific container.
+   */
   public function containersSsh() {
     $this->title('Logging in to PHP container');
     $this->say('This command is not implemented yet. Copy and execute the following:');
     $this->io()->text('docker-compose --file ./vendor/bluesparklabs/spark/docker/docker-compose.d8.yml --project-name \'Spark Example\' exec php bash');
   }
+
+  /**
+   * Limit composer to specific service.
```

Comment:
Minor thing, but I would like to avoid confusion: We're limiting Docker Compose to a service, not Composer. :)